powerquery-docs/connectors/solarwinds-service-desk.md

```diff
@@ -34,41 +32,114 @@ Before you can use this connector to get SolarWinds Service Desk data, you must
 
 ## Connect to SolarWinds Service Desk
 
-To connect to SolarWinds Service Desk:
+The instructions below contain special steps for customers who previously used the BETA SWSD Power BI Connector to create reports and wish to connect those reports with the new out-of-the-box version.
 
-1. In Power BI Desktop, select **Get Data** from **Home**. Select **Other** from the categories on the left, select **SolarWinds Service Desk**, and then select **Connect**.
+1. In Power BI Desktop, select **Get Data** from **Home**. Select **SolarWinds Service Desk**, and then select **Connect**.
 
    ![Screenshot of Get Data from SolarWinds Service Desk.](./media/solarwinds-service-desk/get-data.png)
 
-2. Sign in with the JSON web token you generated as described in prerequisites, and then select **Connect** to verify your access to SolarWinds Service Desk.
+2. Sign in with the JSON web token you generated as described in prerequisites, and then click **Connect** to verify your access to SWSD.
```

Comment:
Change "...and then click **Connect** to..." to "...and then select **Connect** to..." to meet Microsoft style rules.